src/pebble.py

```diff
@@ -35,7 +39,7 @@ def replan_jenkins(
     try:
         jenkins_instance.wait_ready()
         # Tested in integration
-        if state.auth_proxy_integrated:  # pragma: no cover
+        if disable_security:  # pragma: no cover
```

Comment:
I'm wondering if a unit test would be worth it to remove the `no cover` statement even though it is tested in integration tests :) Also up to you!

Reply:
I have to investigate further to understand why this was needed in the first place, I'll resolve this for now and do a follow-up PR to address this.